docs/How-To_Guides/OpenStack/Barbican/index.md

```diff
@@ -0,0 +1,25 @@
+# Using Barbican for secret storage
+
+**[Barbican](https://docs.openstack.org/barbican/latest/)** is
+OpenStack's secret storage facility. In {{extra.brand}}, Barbican is
+supported for the following purposes:
+
+* [Generic secret storage](generic-secret.md),
+* [encryption for persistent volumes](../Cinder/encrypted-volumes.md),
+* [certificate storage for HTTPS load balancers](../Octavia/tls-lb.md).
+
+To manage secrets with Barbican, you will need the `openstack` command
+line interface (CLI), and its Barbican plugin. You can install them
+both with the following commands:
+
+```bash
+pip install python-openstackclient python-barbicanclient
```

Comment:
This is a bad idea to advise people to pip install directly. Let's just say `pip install --user` please.

Reply:
Well if you're not `root`, then `pip install` will fail anyway. Except if you're in a virtualenv, in which case it's perfectly appropriate. No?

Reply:
It works as a non-root user.

Reply:
Correcting myself:

* Prior to pip 20.0.0, running `pip install` as non-`root` would fail because it would attempt to write to an unwritable location.
* [Since pip 20.0.0](https://pip.pypa.io/en/stable/news/#id434), invoking just `pip` as non-`root` is equivalent to invoking `pip --user`.

Which means that

* If you are `root` and want to `pip`-install a package globally, the command as shown is correct.
* If you are not `root` and want to install a package in the user site directory, as of pip 20.0.0 (2+ years ago), the command as shown is correct.
* Regardless of whether you are `root` or not, and you're in a virtualenv and want to `pip`-install a package into that virtualenv, the command as shown is correct.

Thus, I'd prefer to leave it as-is.

Reply:
Fine for me. 
I guess I am old and habits are hard to break.